auslib/web/views/client.py

```diff
@@ -51,9 +51,13 @@ def getQueryFromURL(self, url):
         return query
 
     def get(self, **url):
-        query = self.getQueryFromURL(url)
-        self.log.debug("Got query: %s", query)
-        release, update_type = AUS.evaluateRules(query)
+        try:
+            query = self.getQueryFromURL(url)
+            self.log.debug("Got query: %s", query)
+            release, update_type = AUS.evaluateRules(query)
+        except KeyError:
+            release = None
+
```

Comment:
Actually no..this block was needed for the first version of the fix that I had..